File: Responder/Responder/Commands/HeartbeatResponse.cs
namespace Responder.AppConfig
{
    public class HeartbeatResponse : ICommand
    {
        #region Properties
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        public int Status { get; set; }
        #endregion

        #region Constructors
        public HeartbeatResponse()
        {
            Type = CommandType.HeartbeatResponse;
        }
        #endregion
    }
}

File: Imperium/src/com/imperium/Commands/HeartbeatResponse.java
package com.imperium.Commands;

public class HeartbeatResponse implements ICommand {
    private CommandType type;
    private int length;
    private String status;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public HeartbeatResponse(){
        type = CommandType.HeartbeatResponse;
    }
}
